app/Http/Controllers/TransactionController.php

```diff
@@ -21,8 +23,10 @@ public function export(Transaction $transaction)
         return view('transactions.export', compact('transaction'));
     }
 
-    public function duplicate(Transaction $transaction)
+    public function duplicate(string $transaction)
     {
+        $transaction = Transaction::where('uuid', $transaction)->firstOrFail();
```

Comment:
@winddance for that, there's a better way, see #1

Reply:
@PovilasKorop thanks for your reply. yes i did reviewed others PR and noticed a better approach. appreciated your good time.